okhttp/src/test/java/okhttp3/URLConnectionTest.java

```diff
@@ -2367,6 +2370,71 @@ private void testResponseRedirectedWithPost(int redirectCode, TransferKind trans
     testRedirect(false, "POST");
   }
 
+  class RevertInterceptor implements Interceptor {
+    @NotNull @Override public Response intercept(@NotNull Chain chain) throws IOException {
```

Comment:
nit: we don’t typically use `@NotNull` because everything is not null unless otherwise specified